### PR TITLE
Emojification

### PR DESCRIPTION
**Pull request by AdrianSosic**
Gives our README a shiny touch by adding emojis

### CODE REVIEW

**Comment by AdrianSosic** [comment]
Hi @Scienfitz and @AVHopp, had to do something less serious in between. What do you think, better or worse? Perhaps it helps a little bit to visually parse the bullet list. At least, it gives some glossy look :D

In case you decide to keep it, feel free to look for some better matching alternatives if you want to have a short break from you regular tasks. Of course, we can also drop them again 😄

File: README.md [hunk 1]
```diff
@@ -30,15 +30,15 @@ The Bayesian Back End (**BayBE**) provides a general-purpose toolbox for Bayesia
...
-- Built-in chemical encodings: Improve your campaign with chemical knowledge
-- Single and multiple targets with min, max and match objectives
-- Custom surrogate models: For specialized problems or active learning
-- Hybrid (mixed continuous and discrete) spaces
-- Transfer learning: Mix data from multiple campaigns and accelerate optimization
-- Comprehensive backtest, simulation and imputation utilities: Benchmark and find your best settings
-- Fully typed and hypothesis-tested: Robust code base
-- All objects are fully de-/serializable: Useful for storing results in databases or use in wrappers like APIs
+- ✨ Custom parameter encodings: Improve your campaign with domain knowledge
+- ⚗️ Built-in chemical encodings: Improve your campaign with chemical knowledge
```
Comment (Scienfitz) [inline README.md:34]: suggest 🧪

Comment (AdrianSosic) [reply README.md]: done

File: README.md [hunk 2]
```diff
@@ -30,15 +30,15 @@ The Bayesian Back End (**BayBE**) provides a general-purpose toolbox for Bayesia
...
-- Single and multiple targets with min, max and match objectives
-- Custom surrogate models: For specialized problems or active learning
-- Hybrid (mixed continuous and discrete) spaces
-- Transfer learning: Mix data from multiple campaigns and accelerate optimization
-- Comprehensive backtest, simulation and imputation utilities: Benchmark and find your best settings
-- Fully typed and hypothesis-tested: Robust code base
-- All objects are fully de-/serializable: Useful for storing results in databases or use in wrappers like APIs
+- ✨ Custom parameter encodings: Improve your campaign with domain knowledge
+- 🧪 Built-in chemical encodings: Improve your campaign with chemical knowledge
+- 🔍 Single and multiple targets with min, max and match objectives
```
Comment (Scienfitz) [inline README.md:35]: Suggest 🎯 to symbolize target better

**Review by Scienfitz — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (AdrianSosic) [reply README.md]: Definitely better 👌🏼 changed it

**Review by AVHopp — APPROVED** [review]

**Pull request merged**